pkgs/applications/display-managers/lightdm-slick-greeter/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, vala
+, intltool
+, autoreconfHook
+, wrapGAppsHook
+, lightdm
+, gtk3
+, pixman
+, libcanberra
+, libX11
+, libXext
+, linkFarm
+, lightdm-slick-greeter
+, numlockx
+, numlockSupport ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lightdm-slick-greeter";
+  version = "1.5.9";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "slick-greeter";
+    rev = "1.5.9";
+    sha256 = "sha256-UEzidH4ZWggcOWHHuAclHbbgATDBdogL99Ze0PlwRoc=";
+  };
+
+  nativeBuildInputs = [ pkg-config vala intltool autoreconfHook wrapGAppsHook ];
+  buildInputs =
+    [ lightdm gtk3 pixman libcanberra libX11 libXext ]
+    ++ lib.optionals numlockSupport [ numlockx ];
```

Comment:
Maybe we don't need numlockx in buildInputs at all if it is only called once in `src/slick-greeter.vala`